atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1489,13 +1489,21 @@ int WebContents::GetFrameRate() const {
 }
 
 void WebContents::Invalidate() {
-  if (!IsOffScreen())
-    return;
-
-  auto* osr_rwhv = static_cast<OffScreenRenderWidgetHostView*>(
+  if (IsOffScreen()) {
+    auto* osr_rwhv = static_cast<OffScreenRenderWidgetHostView*>(
       web_contents()->GetRenderWidgetHostView());
-  if (osr_rwhv)
-    osr_rwhv->Invalidate();
+    if (osr_rwhv)
+      osr_rwhv->Invalidate();
+  } else {
+    const auto ownerWindow = owner_window();
+    const auto nativeWindow = ownerWindow ? ownerWindow->GetNativeWindow() :
+                                            nullptr;
+    if (nativeWindow) {
+      const gfx::Rect& bounds = nativeWindow->bounds();
+      nativeWindow->SchedulePaintInRect(
```

Comment:
I originally wanted to do that, but on windows `nativeWindow` is of type `aura::Window` and `SchedulePaint()` is a private member. https://cs.chromium.org/chromium/src/ui/aura/window.h?q=schedulepaint&dr=CSs&l=181

`c:\programming\electron\atom\browser\api\atom_api_web_contents.cc(1502): error C2248: 'aura::Window::SchedulePaint': cannot access private member declared in class 'aura::Window'
c:\programming\electron\vendor\brightray\vendor\download\libchromiumcontent\src\ui\aura\window.h(375): note: see declaration of 'aura::Window::SchedulePaint'
c:\programming\electron\vendor\brightray\vendor\download\libchromiumcontent\src\ui\aura\window.h(69): note: see declaration of 'aura::Window'`